app/services/expenses/RemoveExpensesService.scala

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package services.expenses
+import common.EmploymentToRemove._
+import config.ErrorHandler
+import connectors.{DeleteOrIgnoreExpensesConnector, IncomeSourceConnector}
+import controllers.employment.routes.EmploymentSummaryController
+import models.User
+import models.employment.AllEmploymentData
+import play.api.Logging
+import play.api.mvc.Results.Redirect
+import play.api.mvc.{Request, Result}
+import uk.gov.hmrc.http.HeaderCarrier
+
+import javax.inject.Inject
+import scala.concurrent.{ExecutionContext, Future}
+
+class RemoveExpensesService @Inject()(deleteOrIgnoreExpensesConnector: DeleteOrIgnoreExpensesConnector,
+                                      incomeSourceConnector: IncomeSourceConnector,
+                                      errorHandler: ErrorHandler,
+                                      implicit val ec: ExecutionContext) extends Logging {
+
+  def deleteOrIgnoreExpenses(employmentData: AllEmploymentData, taxYear: Int)(result: Result)
```

Comment:
don't think this service is needed, can call the existing deleteOrIgnoreExpenses method in DeleteOrIgnoreExpensesService.